Formula/gptfdisk.rb

```diff
@@ -27,8 +34,26 @@ def install
   end
 
   test do
-    system "hdiutil", "create", "-size", "128k", "test.dmg"
+    system "dd", "if=/dev/zero", "of=test.dmg", "bs=1m", "count=1"
+    output = shell_output("#{bin}/sgdisk -o test.dmg")
+    assert_match "completed successfully", output
+    output = shell_output("#{bin}/sgdisk -p test.dmg")
+    assert_match "GUID", output
     output = shell_output("#{bin}/gdisk -l test.dmg")
     assert_match "Found valid GPT with protective MBR", output
```

Comment:
```suggestion
    assert_match "Found valid GPT with protective MBR", shell_output("#{bin}/gdisk -l test.dmg")
```
I think you should go ahead and change it (and the previous one) if you agree with me regarding readability.

Reply:
If you don't agree, however, please do keep it as is.

Reply:
Well I'm a homebrew noob (at least when it comes to fixing formulae) so I'll defer to your opinion.

I pushed the change to the PR... I assume that the merge bot squashes multiple commits, or is there something else that needs to be done on my end?

Reply:
It does for PRs involving only one formula, so you don't have to worry about this one.